c_emulator/riscv_sim.c

```diff
@@ -120,6 +120,7 @@ static struct option options[] = {
     {"disable-writable-misa",       no_argument,       0, 'I'},
     {"disable-fdext",               no_argument,       0, 'F'},
     {"mtval-has-illegal-inst-bits", no_argument,       0, 'i'},
+    {"c_hints_expand",              no_argument,       0, 'H'},
```

Comment:
Just a thought.

I'm wondering if this needs to be `"c-hints-expand"`. That way, it uses the same naming convention as the other flags.

Reply:
Oops, sorry, good catch!